PRS4001-5000/PRS4283GabraKr.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
 schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="pers" xml:lang="en" xml:id="PRS4283GabraKr">
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="pers" xml:lang="en" xml:id="PRS4283GabraKr">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Gabra Krǝstos of Daga ʾƎsṭifānos</title>
+            <title>Gabra Krǝstos of Gunda Gunde</title>
```

Comment:
are you sure the two are the same? I do not see the reason for merging the two.

GK of DE was abbot at DE, he is mentioned in the EAe "At that time, the monastery (Daga) seems to have
been especially prosperous. Between 1563 and
1574, mämher Gäbrä Krestos restored the
church with financial support from käntiba
Gäbrä Mikael. Two canvases show that a part
of the murals may have been renewed at this
point (Bosc-Tiessé 2001:8589)."

Reply:
In the original entries created from the EAe spreadsheet,  PRS4283GabraKr was the abbot of Dabra Estifanos, hence this title. But since then, the entry has been edited and used to refer to the abbot of Gunda Gunde (which was originally supposed to be PRS4285GabraKr, who has also been used to refer to the same person, the abbot of Gunda Gunde), so these two need to be merged now, as we have to work with the current status quo, not the original intention behind the records.
I find re-using in such cases relatively dangerous, because it increases the confusion and people might still use the "old" IDs in some unmerged branches. I find it better to completely clear it up.